tests/base/models.py

```diff
@@ -67,7 +67,7 @@ def forward(self, img):
 class TestGAN(LightningModule):
     """Implements a basic GAN for the purpose of illustrating multiple optimizers."""
 
-    def __init__(self, hidden_dim, learning_rate, b1, b2, **kwargs):
+    def __init__(self, hidden_dim=128, learning_rate=0.001, b1=0.5, b2=0.999, **kwargs):
```

Comment:
these could use typehints. Is there an open GH issue for adding typehints to more of the lightning codebase, or to tests specifically?

Reply:
thanks! we had such an issue, but after adding typehints to the majority of the codebase, we closed it. there are only a few spots like this one where we forgot them. fixed now :)